### PR TITLE
Add article about rpm build dependencies

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

### CODE REVIEW

File: content/blog/extracing-rpm-build-depends.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+```
+$ yum-builddep -y ${SPEC_FILE}
+```
+
+## Okay that was too easy, why even write a blog post about this?
+
+Well, my traveler, for all the good that `yum-builddep` does, it has the
+most common flaw of them all: **it's not available everywhere**.
+
+For a tool as good as it is, it's reliance on underlying package managers
```
Comment (jose-bigio) [inline content/blog/extracing-rpm-build-depends.md]: I think it's should be its

Comment (jose-bigio) [reply content/blog/extracing-rpm-build-depends.md]: Never mind I think it reads correctly as is.

Comment (seemethere) [reply content/blog/extracing-rpm-build-depends.md]: switched `it's` to `the`

**Review by jose-bigio — APPROVED** [review]
LGTM

**Pull request merged**